embedchain/config/vectordbs/ChromaDbConfig.py

```diff
@@ -27,9 +28,10 @@ def __init__(
         :type port: Optional[str], optional
         :param chroma_settings: Chroma settings dict, defaults to None
         :type chroma_settings: Optional[dict], optional
+        :param allow_reset: Resets the database. defaults to False
```

Comment:
I'd prefer if you changed the order (I know that could break stuff), but `chroma_settings` is a catch-all solution that should come last.